### DISCUSSION TITLE
Is an internet connection required to use webui?

### DISCUSSION THREAD

**Discussion by devPhases** [oobabooga/text-generation-webui#5811] [General]
Help me figure this one out. I updated, and then the webui stopped working, always crashes after a few minutes. I connected to my limited hotspot to check this github for updates, then went back to try the webui again and it now works. After some testing, it seems that the webui only works when i have internet connected. Is this a new "feature" of gradio? Any way around it?

edit: Nevermind, it still crashes.

**Comment by TheLounger** [comment]
I've been running an offline installation on one of my systems for around 11 months (update it like once a week), works fine.
llama-cpp-python 0.2.59 seems to be broken in some way (segfault), maybe that's the issue?
